BotFramework/Core/Bot.cs

```diff
@@ -113,7 +120,7 @@ private void ProcessMessage(BotEventArgs e)
             if (result.IsFailed)
             {
                 LoggerHolder.Instance.Error(result.ToString());
-                _botProvider.WriteMessage(new BotEventArgs(result.ToString(), e));
+                if (_errorLogToUser) _botProvider.WriteMessage(new BotEventArgs(result.ToString(), e));

```

Comment:
Лучше делать перенос
```suggestion
                if (_errorLogToUser)
                    _botProvider.WriteMessage(new BotEventArgs(result.ToString(), e));

```